### PR TITLE
ucm2: add the ucm support for the mt8195-demo platform

### PR DESCRIPTION
**Pull request by fchiby**

### CODE REVIEW

**Comment by fchiby** [comment]
This use alsa-ucm-conf package to detect audio hardware automatically for supporting all AIoT platforms in one single image. 
This pull request:
    - Add the initial support of alsa ucm for the mt8195-demo platform and enable JACK, I2S and PDM devices. 
    - Add HDMI and DP audio for i1200-demo

File: ucm2/MediaTek/mt8195_demo/HiFi.conf [hunk 1]
```diff
@@ -0,0 +1,131 @@
+SectionVerb {
+	EnableSequence [
+		cset "name='Headset Volume' 0"
```
Comment (perexg) [inline ucm2/MediaTek/mt8195_demo/HiFi.conf:3]: The card initialization should be in the BootSequence section.

Comment (fchiby) [reply ucm2/MediaTek/mt8195_demo/HiFi.conf]: Thanks for the review. I've just pushed a new version which fixing this.

**Comment by perexg** [comment]
Applied with two changes:

- fixed the symlink from conf.d
- removed duplicated Dp device

**Pull request closed without merging**